CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * [BUGFIX] Log the names of services that are not yet running rather than `unsupported value type` when calling `/ready` and some services are not running. #3625
 
 ### Mixin
+* [BUGFIX] Alerts: Fixed `MimirIngesterHasNotShippedBlocks` and `MimirIngesterHasNotShippedBlocksSinceStart` alerts for when Mimir is deployed in read-write or monolithic modes. #3627
```

Comment:
Add a CHANGELOG entry about the new metric too. The new CHANGELOG entry should go above under "Grafana Mimir" section.